<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-V7U200JF" name="fuyoujiankang" tests="2" failures="1" timestamp="30 Nov 2017 06:17:31 GMT" time="19.779" errors="0">
  <testcase name="addSuccessFunction" time="0.208" classname="com.fuyoujiankang.test.testcases.serviceManagement.ComboPage_002_AddSuccessFunction_Test">
    <skipped/>
  </testcase> <!-- addSuccessFunction -->
  <testcase name="addSuccessFunction" time="0.017" classname="com.fuyoujiankang.test.testcases.serviceManagement.ComboPage_002_AddSuccessFunction_Test">
    <failure type="java.lang.AssertionError" message="输入 [18817715180] 到 元素[By.id: username]失败">
      <![CDATA[java.lang.AssertionError: 输入 [18817715180] 到 元素[By.id: username]失败
	at com.fuyoujiankang.test.utils.SeleniumUtil.type(SeleniumUtil.java:168)
	at com.fuyoujiankang.test.pageshelper.LoginPageHelper.typeHospitalLoginInfo(LoginPageHelper.java:26)
	at com.fuyoujiankang.test.testcases.serviceManagement.ComboPage_002_AddSuccessFunction_Test.addSuccessFunction(ComboPage_002_AddSuccessFunction_Test.java:24)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- addSuccessFunction -->
</testsuite> <!-- fuyoujiankang -->
